lib/file_transfer_agent/gcs_util.js

```diff
@@ -69,7 +69,9 @@ function GCSUtil(httpclient, filestream) {
         }
       });
 
-      const storage = new Storage({ interceptors_: interceptors });
+      //TODO: hardcoded region will be replaced in the future
+      const isRegionalUrlEnabled = (stageInfo.region).toLowerCase() === 'me-central2' || stageInfo.useRegionalUrl;
+      const storage = isRegionalUrlEnabled ? new Storage({ interceptors_: interceptors, apiEndpoint: `storage.${stageInfo.region.toLowerCase()}.rep.googleapis.com` }) : new Storage({ interceptors_: interceptors });
```

Comment:
we should support explicit endpoint for GCS escpecially

Reply:
I added the case when the endpoint is specified.